source/developers/plugins/studio-plugins.rst

```diff
@@ -0,0 +1,161 @@
+:is-up-to-date: True
+
+.. index:: Crafter Studio Plugins, Studio Plugins, Plugins
+
+.. _studio-plugins:
+
+======================
+Crafter Studio Plugins
+======================
+
+Crater Studio plugins allows users to replace a piece in Studio or work in conjunction with Studio.
+It is for the authoring environment and can be a separate app that runs outside of Studio UI, or it can be a piece of widget or a sidebar item added inside the Studio UI.
```

Comment:
Studio plugins extend the authoring environment and can be pieces within Studio UI or have their own devoted page inside Studio, in which case plugin authors have a blank canvas to design their full plugin experience.

Stand alone plugins can make use of Studio UI components using various possible mechanisms described below.